<?xml version="1.0" encoding="UTF-8"?>
<testsuites>
    <testsuite name="force.apex" timestamp="2019-12-12T21:02:40-06:00" hostname="https://customization-business-8920-dev-ed.cs94.my.salesforce.com" tests="4" failures="3"  errors="0"  time="0.14">
        <properties>
            <property name="outcome" value="Failed"/>
            <property name="testsRan" value="4"/>
            <property name="passing" value="1"/>
            <property name="failing" value="3"/>
            <property name="skipped" value="0"/>
            <property name="passRate" value="25%"/>
            <property name="failRate" value="75%"/>
            <property name="testStartTime" value="Dec 12, 2019 9:02 PM"/>
            <property name="testExecutionTime" value="0.14 s"/>
            <property name="testTotalTime" value="0.14 s"/>
            <property name="commandTime" value="7.04 s"/>
            <property name="hostname" value="https://customization-business-8920-dev-ed.cs94.my.salesforce.com"/>
            <property name="orgId" value="00D0R0000001hlnUAA"/>
            <property name="username" value="test-tkg6ixvrkmnr@example.com"/>
            <property name="testRunId" value="7070R00000mLvpZ"/>
            <property name="userId" value="0050R000003YmuIQAS"/>
        </properties>
        <testcase name="shouldDeleteForm" classname="FormControllerTest" time="0.05">
            <failure message="System.AuraHandledException: Script-thrown exception"><![CDATA[Class.FormController.deleteForm: line 42, column 1
Class.FormControllerTest.shouldDeleteForm: line 41, column 1]]></failure>
        </testcase>
        <testcase name="shouldDeleteFormResponse" classname="FormControllerTest" time="0.02">
            <failure message="System.QueryException: List has no rows for assignment to SObject"><![CDATA[Class.FormController.deleteForm: line 39, column 1
Class.FormControllerTest.shouldDeleteFormResponse: line 31, column 1]]></failure>
        </testcase>
        <testcase name="shouldGetFormResponses" classname="FormControllerTest" time="0.02">
            <failure message="System.AssertException: Assertion Failed: Expected: 1, Actual: 0"><![CDATA[Class.FormControllerTest.shouldGetFormResponses: line 22, column 1]]></failure>
        </testcase>
        <testcase name="shouldPublishForm" classname="FormControllerTest" time="0.05">
        </testcase>
    </testsuite>
</testsuites>
